ecf/def/wafs.def

```diff
@@ -1,520 +0,0 @@
-    edit QUEUE_ARCH 'dev_transfer'
-    edit PROJENVIR 'DEV'
-    edit PROJ 'WAFS'
-    edit NET 'wafs'
-    edit RUN 'wafs'
-    family 00
-      edit CYC '00'
-      task jwafs_gfs_manager
-        trigger  /00/gfs/jgfs_forecast == active or /00/gfs/jgfs_forecast == complete
-        event 1 release_wafs_upp_anl
-        event 2 release_wafs_upp_000
-        event 3 release_wafs_upp_001
-        event 4 release_wafs_upp_002
-        event 5 release_wafs_upp_003
-        event 6 release_wafs_upp_004
-        event 7 release_wafs_upp_005
-        event 8 release_wafs_upp_006
-        event 9 release_wafs_upp_007
-        event 10 release_wafs_upp_008
-        event 11 release_wafs_upp_009
-        event 12 release_wafs_upp_010
-        event 13 release_wafs_upp_011
-        event 14 release_wafs_upp_012
-        event 15 release_wafs_upp_013
-        event 16 release_wafs_upp_014
-        event 17 release_wafs_upp_015
-        event 18 release_wafs_upp_016
-        event 19 release_wafs_upp_017
-        event 20 release_wafs_upp_018
-        event 21 release_wafs_upp_019
-        event 22 release_wafs_upp_020
-        event 23 release_wafs_upp_021
-        event 24 release_wafs_upp_022
-        event 25 release_wafs_upp_023
-        event 26 release_wafs_upp_024
-        event 27 release_wafs_upp_027
-        event 28 release_wafs_upp_030
-        event 29 release_wafs_upp_033
-        event 30 release_wafs_upp_036
-        event 31 release_wafs_upp_039
-        event 32 release_wafs_upp_042
-        event 33 release_wafs_upp_045
-        event 34 release_wafs_upp_048
-        event 35 release_wafs_upp_054
-        event 36 release_wafs_upp_060
-        event 37 release_wafs_upp_066
-        event 38 release_wafs_upp_072
-        event 39 release_wafs_upp_078
-        event 40 release_wafs_upp_084
-        event 41 release_wafs_upp_090
-        event 42 release_wafs_upp_096
-        event 43 release_wafs_upp_102
-        event 44 release_wafs_upp_108
-        event 45 release_wafs_upp_114
-        event 46 release_wafs_upp_120
-        event 47 release_wafs_gcip_000
-        event 48 release_wafs_gcip_003
-        event 49 release_wafs_grib_006
-        event 50 release_wafs_grib_012
-        event 51 release_wafs_grib_018
-        event 52 release_wafs_grib_024
-        event 53 release_wafs_grib_030
-        event 54 release_wafs_grib_036
-        event 55 release_wafs_grib_042
-        event 56 release_wafs_grib_048
-        event 57 release_wafs_grib_054
-        event 58 release_wafs_grib_060
-        event 59 release_wafs_grib_066
-        event 60 release_wafs_grib_072
-      family upp
-        task jwafs_upp_anl
-          trigger ../jwafs_gfs_manager:release_wafs_upp_anl
-          edit FHR anl
-        task jwafs_upp_f000
-          trigger ../jwafs_gfs_manager:release_wafs_upp_000
-          edit FHR 000
-        task jwafs_upp_f001
-          trigger ../jwafs_gfs_manager:release_wafs_upp_001
-          edit FHR 000
```

Comment:
Is this a typo why UPP 001 didn't go through? Should it be "edit FHR 001"?

Reply:
yes.  Thanks. That was indeed a typo.
`jwafs_upp_f001` ran, but it probably ran as if `FHR = 000`.
Fixed.